docs/antrea-network-policy.md

```diff
@@ -1398,6 +1398,10 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+More generally speaking, it is recommended not to use the FQDN selector for DNS
```

Comment:
```suggestion
More generally speaking, it is not recommended to use the FQDN selector for DNS
```
nit, wording made me think it is the other way around

Reply:
They mean essentially the same thing, but I don't mind switching